Doc/library/functions.rst

```diff
@@ -479,7 +479,7 @@ are always available.  They are listed here in alphabetical order.
 .. function:: enumerate(iterable, start=0)
 
    Return an enumerate object. *iterable* must be a sequence, an
-   :term:`iterator`, or some other object which supports iteration.
+   :term:`iterator`, or some other object that supports iteration.
```

Comment:
How is "that" better than "which"?

Reply:
It's referring to a defining clause

Reply:
"which" can also refer to defining clauses, see: https://dictionary.cambridge.org/grammar/british-grammar/relative-clauses-defining-and-non-defining